public/javascripts/SVValidate/src/Main.js

```diff
@@ -179,7 +179,12 @@ function Main (param) {
             svv.gsvOverlay = new GSVOverlay();
             svv.keyboard = new Keyboard(svv.ui.validation);
             svv.labelVisibilityControl = new LabelVisibilityControl();
-            svv.speedLimit = new SpeedLimit(svv.panorama.getPanorama(), svv.panorama.getPosition, () => false, svv.panoramaContainer);
+            const labelTypeTemp = param.labelList[0].getAuditProperty('labelType');
+            // Checking to see if the labeltype is NoCurbRamp before adding speedlimit.
+            if (labelTypeTemp == 'NoCurbRamp') {
```

Comment:
Since the `labelTypeTemp` variable is going to be a String, you should use `===` instead of `==` in JavaScript.